lib/p2p/Pool.ts

```diff
@@ -215,7 +215,7 @@ class Pool extends EventEmitter {
       this.logger.error('peer error', err);
     });
 
-    peer.once('open', (handshakeState: HandshakeState) => {
```

Comment:
why removing the type?

Reply:
The type is already explicitly defined in Peer.ts:30, so I thought restating it here is unnecessary. The compiler will already know that `handshakeState` is of type `HandshakeState`. The advantage is we don't need to explicitly define the type everywhere we call `peer.once('open'...)`, it will already be defined for us.